src/map/mob.cpp

```diff
@@ -453,6 +453,8 @@ int mob_parse_dataset(struct spawn_data *data)
 struct mob_data* mob_spawn_dataset(struct spawn_data *data)
 {
 	struct mob_data *md = (struct mob_data*)aCalloc(1, sizeof(struct mob_data));
+	// placement new!
```

Comment:
```suggestion
	// placement new
```
I dont really understand why this comment is needed everywhere, but if you add it, please keep it consistent.

Reply:
I have these here as a todo to remove them, although i guess i can just grep for `new(`